projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -6,17 +6,18 @@
   "license": "MIT",
   "scripts": {
     "cy:open": "cypress open",
-    "cy:run": "cypress run --spec cypress/integration/smoke/**/*",
-    "cy:cli": "cypress run",
-    "cy:run:mobile": "cypress run --config numTestsKeptInMemory=1 --spec \"cypress/integration/mobile/**/*\"",
-    "cy:run:regression": "cypress run --config numTestsKeptInMemory=1 --spec \"cypress/integration/regression/**/*\"",
-    "cy:run:all": "cypress run --config numTestsKeptInMemory=1 --spec \"cypress/integration/**/*\"",
-    "cy:run:vendor": "cypress run --config numTestsKeptInMemory=1 --spec \"cypress/integration/vendor/**/*\"",
-    "cy:cds:run:vendor": "cypress run --config numTestsKeptInMemory=1 --spec \"cypress/integration/vendor/cds/**/*\""
+    "cy:run": "cypress run",
+    "cy:run:ci": "cypress run --config-file cypress.ci.json",
+    "cy:run:smoke": "cypress run --spec cypress/integration/smoke/**/*",
```

Comment:
There were some issues with passing spec without `""`, maybe we should keep them enclosed?

Reply:
Nice catch. Done